CONTRIBUTING.md

```diff
@@ -251,46 +251,94 @@ Each library needs to provide a `ApiCompatVersion` property which is set to the
+
+```bash
+nuget sources
+```
+
+#### NuGet Config file
+
+You can add the dev feed to your NuGet.Config file, which can be at the Solution, User, or Computer level. See [NuGet.Config file locations and uses](https://docs.microsoft.com/nuget/consume-packages/configuring-nuget-behavior#config-file-locations-and-uses) to locate your NuGet.Config file.
 
 ```xml
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="Azure SDK for Net Dev Feed" value="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json" protocolVersion="3" />
-    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="Azure SDK for .NET Dev Feed" value="https://aka.ms/azsdk/net/feed/index.json" />
   </packageSources>
+  <disabledPackageSources>
+    <clear />
+  </disabledPackageSources>
 </configuration>
 ```
 
-You can then consume packages from this package source, remember to check the [Include prerelease](https://docs.microsoft.com/nuget/create-packages/prerelease-packages#installing-and-updating-pre-release-packages) box in Visual Studio when searching for the packages. 
+### 2. Find NuGet Package
 
-To see the list of packages you can browse the [dev feed](https://dev.azure.com/azure-sdk/public/_packaging?_a=feed&feed=azure-sdk-for-net) and search for the package you are interested in. Or you can do it from the command line via `nuget.exe list -Prelease -Source "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json"`
+You can use the following options to find the available dev feed packages:
 
-To consume the a dev package set the exact version in your project or to consume the latest dev package set the version to `x.y.z-dev.*` for example:
+1. Search the Azure SDK for .NET Dev Feed: https://aka.ms/azsdk/net/feed
+1. In Visual Studio, use the [Package Manager UI](https://docs.microsoft.com/nuget/create-packages/prerelease-packages#installing-and-updating-pre-release-packages), be sure to check "Include prerelease".
+1. Use the NuGet CLI, for example `nuget list azure.identity -Prerelease -Allversions`
+
+### 3. Reference NuGet Package
+
+Now that you have found the package you want to use, it is time to add it to your project file.
+
+As you can see in the example below, we want to use the `Azure.Data.Tables` version `3.0.0-alpha.*`. By using the `*` in the version number each `dotnet restore` will pull the latest version from the dev feed.
 
 ```xml
 <Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-  </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.2.0-dev.*" />
+    <PackageReference Include="Azure.Data.Tables" Version="3.0.0-alpha.*"/>
   </ItemGroup>
+  <PropertyGroup>
```

Comment:
I don't think we should use this type of reference it is highly recommended that folks use the Nuget.Config.

Reply:
Can you tell me more about that?  I haven't heard that before.